.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
No need to add `package-lock.json` file into `.gitignore` because package-lock.json` must always be commited and pushed to git

Reply:
Done